dpctl/tests/elementwise/test_abs.py

```diff
@@ -22,9 +22,17 @@ def test_abs_out_type(dtype):
             np.dtype("c16"): np.dtype("f8"),
         }
         assert dpt.abs(X).dtype == type_map[arg_dt]
+
+        out = dpt.empty_like(X, dtype=type_map[arg_dt])
+        dpt.abs(X, out)
```

Comment:
I think `out` should not be allowed to be a positional argument in array API. Could you please change it to a keyword argument in the test suite?

Reply:
sure!